demos/topbars.html

```diff
@@ -723,5 +770,18 @@ <h3 class="topbar-logo-text">Gravatar</h3>
     </ul>
   </nav>
 
+  <script>
+  document.addEventListener('click', (event) => {
+    const toggle = event.target.closest('.topbar-toggle');
+
+    if (toggle) {
+      const isExpanded = toggle.getAttribute('aria-expanded') === 'true';
+      const list = toggle.nextElementSibling;
+
+      toggle.setAttribute('aria-expanded', !isExpanded);
+      list.setAttribute('aria-expanded', !isExpanded);
+    }
+  });
+  </script>
```

Comment:
Can you remove this block of code and add another HTML example that has the topbar opened?